.travis.yml

```
@@ -1,24 +1,50 @@
+  - secure: v5mGm2scI7wdxvDfJl9KKs+3JImErlOGntK7fz1fO5EK/PAhFC8KXa4M43zNhnt075JCAufLPPiT6wBfwwgM4QL3YEAvxzD02I1x6RrGr3U8yAwZbrmg7aLC2pSxEuCTZAjxkaGcMOXOTlMqMxBRPsqTcN2gSgKzsdGMSh+pmZ7wOKZAQWh56QkLUmCZ4QbNVeiPS5nMy1guQYADN/fGDMg7c3Q9FLmp4dV8hFFru0RXiIPneVK+Y9vFWPgNGVlgCwIbzZ5yZzcILOz/dMIS/wWDM+hUt0WSd9Px7xsYOvu9xAh7hRLSM/QfJa3XafmmrshrhxZYNwxa7ZjcsJYRYR1e//TbNooxiBqeV7wIdtoJJXvAwA0D69RN/3ek42Cs824bpwi5/erTwNLA6k1nutrAQ00epWYdKH1W5JHeX/ASgV4rRyIkNlmBbQbYApIKPBaJ+l+1iza46V0Lj3zkvyR5Y+lEieeTihd8MLIdWQvWReQtvAArUXgmZqeIHtJDq8HapO9dpkOlREjwha8h4/h8KMJ8AH9KU0ArbvCDq1Lz9GJCoj9+Lp/SvcS2AtRPFTzX1PHDeYWo02vh5UjLv6l9bp2Ot322qb67HLYXqqILZ53Ul4YRt4XFliiHoVxvbnEsCcmhVyDTz6S15wK+JS+mwRF2TbAPWAuj10zRXbA=
+  - secure: 4g6DececfyooNpoOAsoeiek9Y1GMWi/j03z21VN9WzEWtDPmSpjOa3A/ZPm8e88Davftp1dyCTy16bR3PtfyajiixNaTUnOlvyaDYa8fy+xCEIs/zcXDoYvLWaYwAC4/+CFli5hv8HHc6xRfqnZZhiz056qG+JE9lj66yregkrxEgiwtjOOhzx/HOU4sH3z+Q4SWV7wa6F5BiIvUrrAOe2SveWRl7QRtm3vYObGHJ5m8GHFfu2ndx1nIrgkfUd6OJxkFmolEueVQGpCHh1eAgzM9RIhLWc6zQpGhKOC/izvr17lJHr9bFVOMeIFN2fhKL1zKpEYLPQ1OKyQBPRvjVb0J/RnwhE8uWpCzFAfjq580pTyWKMx+uzW6BhLSDLNX6sTneDD1QRaJ3Roe/gEzeO5/Gy0xxnPP87HDaIbFvfF2CCbgbnFi+s8z/kuvL0xFGnqewmvZ49H4DFGx0QQF4UZTAtNTEwQ8B6D0coOfs5GtR55mLptmZjj/c3IiNyqLVNVwLN9lLxF3hIxIleluzwItCFC40W2oUqlhptuT/fhBIYIOtPz5YMxegts/zcZc3q2v36joTUzH3QtdisDawWimbwxtw227csYJDi4rghvhnwYRNPPK0/yWJqveAx8R1tgoUe59TxWgU4xZs5bMUA1KU46ieOIG7w1Q+gO96i4=
 
 services:
 - docker
 
 script:
-- docker build --target production --tag todo-app:$VERSION .
-- docker build --target test --tag todo-app-test:$VERSION .
-- docker run --rm -e FLASK_APP -e FLASK_ENV -e TRELLO_BASE_URL -e TRELLO_KEY -e TRELLO_TOKEN todo-app-test:$VERSION tests
-- docker run --rm -e FLASK_APP -e FLASK_ENV -e TRELLO_BASE_URL -e TRELLO_KEY -e TRELLO_TOKEN todo-app-test:$VERSION tests_e2e
+- docker build --target production --tag $DOCKER_NAMESPACE/$DOCKER_APP .
+- docker tag $DOCKER_NAMESPACE/$DOCKER_APP $DOCKER_NAMESPACE/$DOCKER_APP:$TRAVIS_COMMIT
+- docker build --target test --tag $DOCKER_NAMESPACE/$DOCKER_APP-test .
+- docker run --rm -e FLASK_APP -e FLASK_ENV -e TRELLO_BASE_URL -e TRELLO_KEY -e TRELLO_TOKEN $DOCKER_NAMESPACE/$DOCKER_APP-test tests
+- docker run --rm -e FLASK_APP -e FLASK_ENV -e TRELLO_BASE_URL -e TRELLO_KEY -e TRELLO_TOKEN $DOCKER_NAMESPACE/$DOCKER_APP-test tests_e2e
+- echo $DOCKERHUB_PWD | docker login --username $DOCKER_NAMESPACE --password-stdin;
+- docker push $DOCKER_NAMESPACE/$DOCKER_APP:$TRAVIS_COMMIT;
+- if [ "$TRAVIS_BRANCH" = "master" ]; then docker push $DOCKER_NAMESPACE/$DOCKER_APP; fi
```

A slightly more condensed format: 
```suggestion
- [[ "$TRAVIS_BRANCH" = "master" ]] && docker push $DOCKER_NAMESPACE/$DOCKER_APP
```

---

I could not seem to get this to work. The "&&" was treated as a special char. Does it need to be escaped in some way ?